src/components/Questionnaire/QuestionTypes/ConsentQuestion.tsx

```
@@ -0,0 +1,405 @@
+
+import CareIcon from "@/CAREUI/icons/CareIcon";
+
+import { Button } from "@/components/ui/button";
+import { DatePicker } from "@/components/ui/date-picker";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Input } from "@/components/ui/input";
```

ESLint

[error] 'Input' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)